### DISCUSSION TITLE
Add precipitation sensor device class

### DISCUSSION THREAD

**Discussion by epenet** [home-assistant/architecture#816] [Entity Models] [answered]
## Proposal

Add a `sensor` `device_class` `accumulated_precipitation` for precipitation to home assistant core and frontend (+ documentation) with supported units:
- `mm`
- `in`

This sensor device class would be useful for most integrations providing weather data sensors.

The unit of a `accumulated_precipitation` sensor shall be configurable by the user in the same way as temperature and pressure sensors etc.

Implementation PR: https://github.com/home-assistant/core/pull/81145

**Comment by emontnemery** [comment, marked as answer]
The proposal is accepted and implemented in core PR https://github.com/home-assistant/core/pull/81145
